views/add_space.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Makers BnB</h1>
+<h2>Add a Space</h2>
+<form method="post" action="/add-space">
+  <%= "Space Name: " %><input type="text" name="space_name" placeholder="Space Name"/><br><br>
+  <%= "Description: " %><input type="text" name="description" placeholder="Description"/><br><br>
```

Comment:
This description could be a `textarea` instead of an `input`, which makes it render a bit nicer on the page. You can also set a maximum amount of characters that the user can input by adding a `maxlength=500` tag to the `textarea`.